cluster-sync/clean.sh

```diff
@@ -56,7 +56,7 @@ set +e
 _kubectl patch cdi ${CR_NAME} --type=json -p '[{ "op": "remove", "path": "/metadata/finalizers" }]'
 set -e
 
-if [ "${CDI_CLEAN}" == "all" ] && [ -f "${OPERATOR_MANIFEST}" ]; then
+if [ -f "${OPERATOR_MANIFEST}" ]; then
```

Comment:
A little worried about this, maybe there was a good reason to keep the operator around..

Reply:
So Arnon did a clean of just cdi or just the test manifests or both. And I think in some of those cases we want to keep the operator around.

Reply:
Does this effectively mean, that when I am developing CDI operator changes and running `cluster-sync`, I am not getting the latest operator build?

Reply:
No a cluster-sync will delete everything, but a cluster-sync-test-infra will keep cdi around and remove the test infra, and cluster-sync-cdi will remove cdi (including operator) and keep the test infra.

Reply:
`cluster-sync-cdi` will not removed the operator unless Im missing something (note `"${CDI_CLEAN}" == "all"`)

Reply:
Would really love a second look at this because this is probably the most breaking change in this PR

Reply:
@awels @arnongilboa

Reply:
actually, we don't need this, the reason was elsewhere https://github.com/kubevirt/containerized-data-importer/pull/2480/files#r1045631832